libs/feature/record/src/lib/data-view-share/data-view-share.component.html

```diff
@@ -0,0 +1,23 @@
+<div *ngIf="wcEmbedderBaseUrl" class="container-lg px-5 my-1 lg:mx-auto">
```

Comment:
If the embedder url is not provided we can still show the webcomponent export, right?

Reply:
I was wondering what would make most sense and rather saw the display of both together. If we want the `data-view-web-component` also without the `data-view-permalink` I think I'll move the decision whether to display the `data-view-share` (with both components) or only the `data-view-web-component` to the `record-metadata` based on `*ngIf="wcEmbedderBaseUrl"`. This will prevent to display the "Share" tab for the `data-view-permalink` if it's not activated.

Reply:
So I added 1a15c881395e13a660e456d7a93667f8fbf265cc for this, encapsulating the logic in the `data-view-share` in the end as I think displaying the tab button as a "label" makes sense as well.